src/MarkdownTextInput.tsx

```diff
@@ -9,6 +9,7 @@ type MarkdownStyle = MarkdownTextInputDecoractorView.MarkdownStyle;
 
 const FONT_FAMILY_MONOSPACE = Platform.select({
   ios: 'Courier',
+  macos: 'Courier',
```

Comment:
```suggestion
  macos: 'SF Mono',
```

Can we use `SF Mono` as default for macOS? 
* https://developer.apple.com/fonts/
* https://en.wikipedia.org/wiki/San_Francisco_(sans-serif_typeface)

Reply:
Sure, thanks for this suggestion! Do we want to stick with `Courier` on iOS though?

Reply:
Done a bit of checking, and looks like SF fonts are not included by default to this day on both iOS and macOS. In this case `Menlo` or `Monaco` could be a viable alternative:
* https://developer.apple.com/fonts/system-fonts/
* https://en.wikipedia.org/wiki/Menlo_(typeface)
* https://en.wikipedia.org/wiki/Monaco_(typeface)